shell/browser/api/electron_api_session.h

```diff
@@ -178,6 +208,16 @@ class Session : public gin::Wrappable<Session>,
   v8::Global<v8::Value> service_worker_context_;
   v8::Global<v8::Value> web_request_;
 
+#if BUILDFLAG(ENABLE_ELECTRON_EXTENSIONS)
+  using GetTabHandler =
+      base::RepeatingCallback<v8::Local<v8::Value>(gin::Handle<WebContents>)>;
```

Comment:
No need to explicitly wrap with `gin::Handle`, the gin converters will handle that automatically.
```suggestion
      base::RepeatingCallback<v8::Local<v8::Value>(WebContents*)>;
```

Reply:
I think I got these errors by this:
![image](https://user-images.githubusercontent.com/11065386/111153181-eb568480-8591-11eb-9888-2ba19a6ba751.png)